packages/compiler/src/init/init.ts

```diff
@@ -194,11 +217,12 @@ async function confirm(message: string): Promise<boolean> {
 async function downloadTemplates(
   host: CompilerHost,
   templatesUrl: TemplatesUrl
-): Promise<Record<string, InitTemplate>> {
+): Promise<Record<string, any>> {
```

Comment:
```suggestion
): Promise<Record<string, unknown>> {
```
unknown better as it force to make sure validation is done later instead of just assuming we can do anything on the type